parsec/serde/fields.py

```diff
@@ -512,3 +519,23 @@ def _deserialize(
 
 
 SequesterEncryptionKeyDer = SequesterEncryptionKeyDerField
+
+
+class PkiEnrollmentSubmitPayloadField(Field[_PkiEnrollmentSubmitPayload]):
+    def _serialize(
+        self, value: _PkiEnrollmentSubmitPayload | None, attr: Any, obj: Any
```

Comment:
Isn't `attr` a `str`?
```
Help on function _serialize in module marshmallow.fields:

_serialize(self, value, attr, obj)
    Serializes ``value`` to a basic Python datatype. Noop by default.
    Concrete :class:`Field` classes should implement this method.

    Example: ::

        class TitleCase(Field):
            def _serialize(self, value, attr, obj):
                if not value:
                    return ''
                return unicode(value).title()

    :param value: The value to be serialized.
    :param str attr: The attribute or key on the object to be serialized.
    :param object obj: The object the value was pulled from.
    :raise ValidationError: In case of formatting or validation failure.
    :return: The serialized value
```